lib/containers/artist.js

```diff
@@ -37,11 +37,19 @@ class Artist extends React.Component {
   render() {
     const windowDimensions = Dimensions.get('window');
     const commonPadding = windowDimensions.width > 700 ? 30 : 20;
+    const tabs = ['ABOUT'];
+    debugger;
+    if (this.props.artist.counts.artworks) {
+      tabs.push('WORKS');
+    }
+    if (this.props.artist.partner_shows.length) {
+      tabs.push('SHOWS');
```

Comment:
Good question, I think you have to ask the analytics people for help on that one.